### PR TITLE
Merge `develop-tab` into `develop` 

### PR DESCRIPTION
**Pull request by lpugin**
See corresponding PRs

### CODE REVIEW

**Comment by craigsapp** [comment]
Are there any sample encodings in the test suite (https://www.verovio.org/test-suite.xhtml)?

**Comment by lpugin** [comment]
Not yes, but it would be good to add some. We need to wait until the [tablature changes](https://github.com/music-encoding/music-encoding/pull/1376) are merged, though.

File: src/view_page.cpp [hunk 1]
```diff
@@ -1285,13 +1283,22 @@ void View::DrawStaff(DeviceContext *dc, Staff *staff, Measure *measure, System *
...
+void View::DrawStaffLines(DeviceContext *dc, Staff *staff, StaffDef *staffDef, Measure *measure, System *system)
 {
     assert(dc);
     assert(staff);
     assert(measure);
     assert(system);
 
+    // If German lute tablature the default is @lines.visible="false", but setting @lines.visible="true"
+    // will draw the staff lines.
+    bool gltLines = (staff->IsTabLuteGerman() && staffDef->GetLinesVisible() == BOOLEAN_true);
```
Comment (lpugin) [inline src/view_page.cpp:1295]:
```suggestion
    bool gltLines = (staff->IsTabLuteGerman() && staffDef->GetLinesVisible() != BOOLEAN_true);
```

**Comment by musicog** [comment]
Test-suite sample encodings are made available through [PR #110 on the verovio.org repository.](https://github.com/rism-digital/verovio.org/pull/110)

**Pull request merged**